cluster/src/main/java/com/hartwig/pipeline/tertiary/orange/Orange.java

```diff
@@ -270,11 +257,15 @@ public OrangeOutput persistedOutput(final SomaticRunMetadata metadata) {
 
     @Override
     public List<AddDatatype> addDatatypes(final SomaticRunMetadata metadata) {
-        final String orangePdf = metadata.tumor().sampleName() + ORANGE_OUTPUT_PDF;
-        final String orangeJson = metadata.tumor().sampleName() + ORANGE_OUTPUT_JSON;
-        return List.of(new AddDatatype(DataType.ORANGE_OUTPUT_JSON,
-                        metadata.barcode(),
-                        new ArchivePath(Folder.root(), namespace(), orangeJson)),
-                new AddDatatype(DataType.ORANGE_OUTPUT_PDF, metadata.barcode(), new ArchivePath(Folder.root(), namespace(), orangePdf)));
+        if (includeGermline) {
```

Comment:
This is the wrong way around. We want to attach the datatypes to the complete orange output with germline.